app.js

```diff
@@ -41,63 +40,9 @@ const User = mongoose.model("User", userSchema);    //creating model according t
 // user.save();
 
 
-
-function findUserName(id){
-  User.findById(id, 'Profile',function(err, member){
-    if(err){
-      console.log(err);
-    }else{
-      console.log(member.Profile.name);
-    }
-  })
-};
-
-
-
-
-function findUserEmail(id){
-  User.findById(id, 'Profile',function(err, member){
-    if(err){
-      console.log(err);
-    }else{
-      console.log(member.Profile.email);
-    }
-  })
-};
-
-function findUserPassword(id){
-  User.findById(id, 'Profile',function(err, member){
-    if(err){
-      console.log(err);
-    }else{
-      console.log(member.Profile.password);
-    }
-  })
-};
-
-
-function findUserId(name,email){
-  User.findOne({"Profile.email": email, "Profile.name": name}, 'Profile',function(err, member){
-    if(err){
-      console.log(err);
-    }else{
-      console.log(member._id);
-    }
-  })
-};
-
-function deleteUser(id_delete){
-  User.findByIdAndDelete(id,function(err){
-    if(err){
-      console.log(err);
-    }else{
-      console.log("The user was successfully deleted.");
-    }
-  })
-};
-id_d="";
-findUserName(id);
-findUserEmail(id);
-findUserPassword(id);
-findUserId(name,email);
-deleteUser(id_d);
+//UserScripts.createUser("Adrien", "myemail@gmail.com","safe_password" , User);
+UserScripts.findUserName(id, User);
+UserScripts.findUserEmail(id, User);
+UserScripts.findUserPassword(id, User);
+UserScripts.findUserId("myemail@gmail.com", User);  //Doesn't work in quick succession with createUser
```

Comment:
Strangely this doesn't function isn't able to execute when it is called right after a user has been created. Must be a synchronicity problem but not essential for now.